### PR TITLE
feat:add info about biratnagar

### PR DESCRIPTION
**Pull request by aayush105**
## What kind of change does this PR introduce?

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added a new city
- [ ] Other(s)

### CODE REVIEW

File: data/borderlines.json [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+        "value": 77,
+        "unit": "km²"
+      },
+      "elevation": {
+        "value": 73.728,
+        "unit": "meters"
+      },
+      "landmarks": [
+        {
+          "name": "Baraha Kshetra",
```
Comment (thenishantsapkota) [inline data/borderlines.json:23]: I don't think this landmark falls within Biratnagar

File: data/borderlines.json [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+          "latitude": 26.83,
+          "longitude": 87.18
+        },
+        {
+          "name": "Koshi Tappu Wildlife Reserve",
```
Comment (thenishantsapkota) [inline data/borderlines.json:28]: Same for this, I don't think this falls within biratnagar

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/borderlines.json (hunk 2 above)
Comment (aayush105) [reply data/borderlines.json:28]: I will check it and correct it.

**Comment by aayush105** [comment]
> Also please add more landmarks here. Atleast 5 per city

Sure I will add 2 more.

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution @aayush105!

**Pull request merged**